package.json

```diff
@@ -31,7 +31,7 @@
     "angular-animate": "^1.4.8",
     "angular-resource": "^1.4.8",
     "d3fc": "^5.2.0",
-    "BitFlux": "ScottLogic/BitFlux#cf81eab098cf6c6f4a4bd36269d9cfe21d4b449b",
+    "BitFlux": "bjedrzejewski/BitFlux#dddca873ab952c7d9f792c54155b4b73594e7ab3",
```

Comment:
The current BitFlux we're using is https://github.com/ScottLogic/BitFlux/commit/cf81eab098cf6c6f4a4bd36269d9cfe21d4b449b, so your change will actually be updated the version of BitFlux too. Do we want that?